doc/getting_started/setup_fpga.md

```diff
@@ -399,7 +399,7 @@ Then a connection between OpenOCD and GDB may be established with:
 ```console
 cd $REPO_TOP
 riscv32-unknown-elf-gdb -ex "target extended-remote :3333" -ex "info reg" \
-  "$(./bazelisk.sh outquery --config=riscv32 //sw/device/tests:uart_smoketest_prog_fpga_cw310_test_rom.elf)"
+  "$(./bazelisk.sh outquery --config=riscv32 //sw/device/tests:uart_smoketest_prog_fpga_cw310.elf)"
```

Comment:
Yes, this target is correct. Just for my curiosity, what does outquery do?

Reply:
Outquery is a convenience feature that attempts to print a path to the given target's output file (assumes there's only one). It's implemented directly in [`bazelisk.sh`](https://cs.opensource.google/opentitan/opentitan/+/master:bazelisk.sh;l=122;drc=ca65fe29bdb810ad8b02f0c8dda0dfc83c95ecf7). Be forewarned, it doesn't always give the expected result, e.g. targets in the `@bitstreams//` workspace.

Reply:
I should also add a prompt to build the elf file so there's something at the expected path.